### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by emaaForlin**
This modification adds a large number of phrases translated into Spanish, which although it is not the most used language in the K8S world, can be helpful for a lot of Hispanics who are learning and does not speak English. That was my case and i hope it helps

### CODE REVIEW

**Comment by medyagh** [comment]
thank you @emaaForlin  for this contribution, would u plz sign the CLA so we could accept this contribution ?
https://identity.linuxfoundation.org/projects/cncf

**Comment by emaaForlin** [comment]
Okey, i already signed the CLA so i hope this PR will be accepted

File: translations/es.json [hunk 1]
```diff
@@ -1,181 +1,181 @@
...
+	"Automatically selected the {{.driver}} driver. Other choices: {{.alternates}}": "Controlador {{.driver}} seleccionado automáticamente. Otras opciones: {{.alternates}}",
+	"Available Commands": "Comandos disponibles",
+	"Basic Commands:": "Comandos basicos:",
+	"Because you are using a Docker driver on {{.operating_system}}, the terminal needs to be open to run it.": "Porque estás usando controlador Docker en {{.operating_system}}, la terminal debe abrirse para ejecutarlo.",
+	"Bind Address: {{.Address}}": "Dirección de enlace: {{.Address}}",
+	"Both driver={{.driver}} and vm-driver={{.vmd}} have been set.\n\n    Since vm-driver is deprecated, minikube will default to driver={{.driver}}.\n\n    If vm-driver is set in the global config, please run \"minikube config unset vm-driver\" to resolve this warning.\n\t\t\t": "Ambos driver={{.driver}} y vm-driver={{.vmd}} han sido establecidos.\n\n	vm-driver y es obsoleto, el por defecto de minikube será driver={{.driver}}.\n\n Si vm-driver está establecido en la configuracion global, ejecuta \"minikube config unset vm-driver\" para resolver esta advertencia.\n\t\t\t",
```
Comment (sharifelgamal) [inline translations/es.json:53]: This line has a tab character at the `\n\n	vm-driver y es obsoleto,`. Changing that tab character to spaces after the two newlines will make this valid json. Once that's fixed we can accept this PR.

**Comment by sharifelgamal** [comment]
Ah sorry, I was clear enough with my instructions. we need to keep the '\t' character as \t, not as tab characters, but there is one actual tab in the line `"Ambos driver={{.driver}} y vm-driver={{.vmd}} han sido establecidos.\n\n	vm-driver y es obsoleto, el por defecto de minikube será driver={{.driver}}.\n\n Si vm-driver está establecido en la configuracion global, ejecuta \"minikube config unset vm-driver\" para resolver esta advertencia.\n\t\t\t"`. 

Copy paste the following and paste it over that line (and restore the \t characters) and everything will be perfect:
```
"Ambos driver={{.driver}} y vm-driver={{.vmd}} han sido establecidos.\n\n vm-driver y es obsoleto, el por defecto de minikube será driver={{.driver}}.\n\n Si vm-driver está establecido en la configuracion global, ejecuta \"minikube config unset vm-driver\" para resolver esta advertencia.\n\t\t\t"
```

**Comment by emaaForlin** [comment]
Okey, i think all the issues are already fixed, but if you find more errors, please let me know.

**Review by sharifelgamal — APPROVED** [review]

**Comment by sharifelgamal** [comment]
Thank you for your contribution!

¡Gracias por tu contribución!

**Pull request merged**